common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -760,6 +835,7 @@ private AuthenticationConstants() {
          * The value for pkeyauth redirect.
          */
         public static final String PKEYAUTH_REDIRECT = "urn:http-auth:PKeyAuth";
+        public static final String CLIENT_TLS_REDIRECT = "urn:http-auth:PKeyAuth";
```

Comment:
This matches the constant declared directly above -- should we just use one of these?

Reply:
removed the "CLIENT_TLS_REDIRECT "